WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderPostActions.java

```diff
@@ -222,9 +222,20 @@ private static boolean handlePostLikes(final ReaderPost post, JSONObject jsonPos
      * similar to updatePost, but used when post doesn't already exist in local db
      **/
     public static void requestPost(final long blogId,
-                                   final long postId,
-                                   final ReaderActions.OnRequestListener requestListener) {
-        String path = "read/sites/" + blogId + "/posts/" + postId + "/?meta=site,likes";
+            final long postId,
+            final ReaderActions.OnRequestListener requestListener) {
+        requestPost(false, blogId, postId, requestListener);
+    }
+
+    /**
+     * similar to updatePost, but used when post doesn't already exist in local db
+     **/
+    public static void requestPost(final boolean isFeed,
```

Comment:
Similar to my earlier comment, I'd prefer to see this as `requestBlogPost` and `requestSitePost`.